catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/SubjectCommands.java

```diff
@@ -33,72 +35,93 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(SubjectCommands.class);
 
+    private final Security security;
+
     @Option(name = "--user", required = false, aliases = {
             "-u"}, multiValued = false, description = "Run command as a different user")
     protected String user = null;
 
+    protected SubjectCommands() {
+        this(Security.getInstance());
+    }
+
+    SubjectCommands(Security security) {
```

Comment:
No, just a unit testing trick :-) It was either that, having a single constructor taking in a `Security` object, using a factory method, adding a package-private `setSecurity()` or using PowerMock. Felt to me like this was the most elegant solution here given all the existing sub-classes, but I'm open to discussion.